app/uk/gov/hmrc/crdlcache/repositories/CustomsOfficeListsRepository.scala

```diff
@@ -211,13 +212,7 @@ class CustomsOfficeListsRepository @Inject() (val mongoComponent: MongoComponent
       )
       .skip((pageNum - 1) * pageSize)
       .limit(pageSize)
-      .map(office =>
-        CustomsOfficeSummary(
-          CustomsOfficeSummary.getReferenceNumber(office),
-          CustomsOfficeSummary.getCountryCode(office),
-          CustomsOfficeSummary.getUsualName(office)
-        )
-      )
+      .map(office => Json.parse(office.toJson()).as[CustomsOfficeSummary]())
```

Comment:
Hmm could this work without the `map` by changing the `find[Document]` above to `find[CustomsOfficeSummary]`?

Reply:
Not on it's own at least, throws `exception=[org.bson.codecs.configuration.CodecConfigurationException: Can't find a codec for CodecCacheKey{clazz=class uk.gov.hmrc.crdlcache.models.CustomsOfficeSummary`

Reply:
Ah, we can probably fix that by using the [extraCodecs](https://github.com/hmrc/hmrc-mongo?tab=readme-ov-file#playmongorepository) property of the PlayMongoRepository, but it's not essential

Reply:
Got time to look at this, changes pushed up.